### PR TITLE
Extract python to be gears plugin

### PR DESCRIPTION
**Pull request by MeirShpilraien**

### CODE REVIEW

File: plugins/python/redisgears_python.c [hunk 1]
```diff
@@ -5544,8 +5546,8 @@ static int PythonConfig_GetStrVal(const char* val, const char* defaultVal, const
 #define ForeceDownloadDepsOnEnterpriseConfigName "ForeceDownloadDepsOnEnterprise"
 #define PythonAttemptTracebackConfigName "PythonAttemptTraceback"
 #define PythonInstallReqMaxIdleTimeConfigName "PythonInstallReqMaxIdleTime"
-#define DependenciesUrlConfigName "DependenciesUrl"
-#define DependenciesSha256ConfigName "DependenciesSha256"
+#define GearsPythonUrlConfigName "GearsPythonUrl"
+#define GearsPythonSha256ConfigName "GearsPythonSha256"
```
Comment (MeirShpilraien) [inline plugins/python/redisgears_python.c:5550]: The name itself can not be changed cause it breaks backword compatability.

File: leakcheck.supp [hunk 2]
```diff
@@ -1,3 +1,10 @@
+{
```
Comment (MeirShpilraien) [inline leakcheck.supp:1]: Why is this needed?

File: plugins/python/redisgears_python.c (hunk 1 above)
Comment (rafie) [reply plugins/python/redisgears_python.c:5550]: I'll revert that.

File: leakcheck.supp (hunk 2 above)
Comment (rafie) [reply leakcheck.supp:1]: It pops up in macOS. Just to clear noise. It's not essential, I can remove it.

Comment (MeirShpilraien) [reply leakcheck.supp:1]: But maybe its not noise?

Comment (rafie) [reply leakcheck.supp]: That's possible. I can reproduce it and we'll look into it.

**Review by rafie — APPROVED** [review]
